Palindrome.cs

```diff
@@ -0,0 +1,21 @@
+using System;  
+  public class PalindromeExample  
+   {  
+     public static void Main(string[] args)  
+      {  
+          int n,r,sum=0,temp;    
```

Comment:
Good and consise, traight to the point, however it doesn't hurt to use more lines here. What does n and r stand for?

Reply:
```suggestion
          int n, r;
          int sum=0;
          int temp;    
```